ImperatorToCK3/Data_Files/configurables/culture_map.txt

```diff
@@ -24,10 +51,10 @@ link = { ck3 = ethiopian imp = aksumite }
 
 ## Anatolian
 link = { ck3 = armenian imp = armenian }
-link = { ck3 = cappadocian imp = cappadocian imp = lycaonian imp = cilician imp = isaurian }
-link = { ck3 = phrygian imp = phyrigian imp = pamphylian imp = lycian }
-link = { ck3 = pontic imp = pontic imp = paphlagonian }
-link = { ck3 = ionic imp = carian imp = mysian imp = lydian }
+link = { ck3 = luwian imp = carian imp = lycian imp = isaurian imp = pisidian }
+link = { ck3 = lydian imp = lydian }
+link = { ck3 = hittite imp = lycaonian imp = cilician imp = pamphylian imp = cappadocian imp = pontic imp = paphlagonian imp = milyadian imp = oroandian imp = cabalian imp = lalasian imp = cennatean imp = morimenian imp = cataonian }
```

Comment:
Isn't Hittite a long-dead culture?

Reply:
Yes, but it seems that those Anatolians in I:R are close descendants to those Hittites and it is also a less confusing name than Greek-sounding regional ones or the already in-use Anatolian.

Reply:
https://en.wikipedia.org/wiki/Anatolian_peoples#Iron_Age

> Following the Bronze Age collapse, a number of Neo-Hittite petty kingdoms survived until about the 8th century BC. Later in the Iron Age, Anatolian languages were spoken by the Lycians, Lydians, Carians, Pisidians and others. These languages were mostly extinct in the Hellenistic period, by the 3rd century BC, although late survival of some remnants is possible, the Isaurian language may have survived into the Late Antiquity, with funerary inscriptions recorded for as late as the 5th century AD.

As I said, Anatolian is already used in other converters to mean "Greek, but in Asia Minor", so I resurrected Hittite as a name for the main chunk and gave Luwian and Lydian their own cultures because they speak a different language than Hittite.